### ISSUE TITLE
Feature Request - Photo upload on mobile apps (Android and iOS)

### ISSUE THREAD

**Issue by stephbagamasbad** [gitterHQ/gitter#854]

**Comment by trevorah** [comment]
I expect @suprememoocow to coming in with some `+1`'s soon...

**Comment by danfiscus** [comment]
:+1:

**Comment by TheJaredWilcurt** [comment]
:+1:

**Comment by thesneakyp** [comment]
:+1:

**Comment by esden** [comment]
👍

**Comment by kbob** [comment]
👍

**Comment by savek-cc** [comment]
+1

**Comment by TheDiamondYT1** [comment]
:+1:

**Comment by TheDiamondYT1** [comment]
This should be implemented in both the mobile apps and the mobile site.

**Comment by revisualize** [comment]
👍

**Comment by scanlime** [comment]
👍⚡️🔥🐝 This would be really really helpful!
Especially for hardware developers it's super useful to share photos of things in the physical world, which usually come from my mobile these days. As it is, this requires sync'ing photos to a PC and uploading from there. Big opportunity to make gitter even better for IoT developers and makers!

**Comment by grahamperrin** [comment]
Yep, it would be nice to have Gitter as a target amongst share options. 

As a workaround I can post a link e.g. at https://gitter.im/gitterHQ/sandbox?at=58ea0c494cb8d0917372e754 but that's good only if what's to be _represented_ is already hosted by a third party service. 

Sometimes it's preferable to use Gitter for the hosting.

**Comment by alfiepates** [comment]
Yeah, posting images is pretty low-friction in the desktop client, it'd be wonderful if this was equally as low-friction from mobile (especially as I do most of my gittering from mobile these days)
